deliver/lib/deliver/upload_metadata.rb

```diff
@@ -183,6 +197,17 @@ def load_from_filesystem(options)
         UI.message("Loading '#{path}'...")
         options[key] ||= File.read(path)
       end
+
+      # Load review information
+      options[:app_review_information] ||= {}
+      REVIEW_INFORMATION_VALUES.values.each do |option_name|
+        path = File.join(options[:metadata_path], REVIEW_INFORMATION_DIR, "#{option_name}.txt")
+        next unless File.exist?(path)
+        next if options[:app_review_information][option_name]
```

Comment:
Oops, my code is wrong.
I took on your suggestion.

https://github.com/fastlane/fastlane/pull/8175/commits/2b5e6d14df6443a13cfe65629d943ae93f62299b